pages/give-devices.js

```diff
@@ -55,98 +109,96 @@ export default function GiveDevicesPage() {
       <Navbar />
       <main>
         <div className="m-auto w-3/4 p-10 my-10 bg-green-300 border border-gray-300">
-          <div className="bg-gray-100 flex justify-center mb-5">
-            <p className="mt-1 text-xl text-black-600">
-              {
-                'This section is still being worked on, please contact us at the following email instead: '
-              }
-              <a
-                className="mt-1 text-xl underline"
-                href="mailto:hi@mhfcares.org"
-              >
-                {'hi@mhfcares.org'}
-              </a>
-            </p>
-          </div>
           <div className="mt-10 sm:mt-0">
             <div className="md:grid md:grid-cols-3 md:gap-6">
               <div className="md:col-span-1">
                 <div className="px-4 sm:px-0">
                   <h3 className="text-lg font-medium leading-6 text-gray-900">
-                    {'Personal Information'}
+                    {'Laptop Donations for Families'}
                   </h3>
                   <p className="mt-1 text-sm text-gray-600">
-                    {'Use a permanent address where you can receive mail.'}
+                    {`Mail us your used or new laptop. We'll set up the donated laptops in our public computer room for the families to use for free. Some of the donated laptops will go to the families who don't have home computers, or who don't have enough computers for all their children to use.`}
                   </p>
                 </div>
               </div>
               <div className="mt-5 md:mt-0 md:col-span-2">
                 <div className="card bordered">
                   <div className="card-body bg-white">
                     <form onSubmit={handleSubmit}>
+                      <h3 className="text-lg font-medium leading-6 text-red-900">
+                        {'Required fields are marked with *'}
+                      </h3>
                       <InputFormControl
                         id="name"
-                        label="Name"
+                        label="Name *"
```

Comment:
We'll want to replace the manually-coded * markings with `required` attributes later, but this is okay for now. We'll iterate.